core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -190,7 +190,7 @@ object Property extends Enumeration {
 
   val PlanCaching: SQLValue[Boolean] = SQLVal[Boolean](
     s"${Constant.PROPERTY_PREFIX}sql.planCaching",
-    "Property to set/unset plan caching", Some(true))
+    "Property to set/unset plan caching", Some(false))
```

Comment:
This change needs to be documented. And recommendations for users under which conditions should planCaching be used, where it is useful etc.

Reply:
A documentation task is added for this https://jira.snappydata.io/browse/SNAP-2855